hack/generate-yamls.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```

Comment:
Unless you need generate-yamls.sh for something else, I don't think you need the extra complexity of having this separate script for a single `ko resolve`.

Reply:
I tend to keep this file, because I cannot run release.sh locally, but I can run all the logic of generate-yamls.sh to better locate the possible issues.

Reply:
Yes, you can. `release.sh --nopublish --notag`

Reply:
Thanks for the great params. Well, still want to keep it, since this is just the beginning of generating yaml files. There will be more coming as we expected.